src/main/java/io/github/thebusybiscuit/slimefun4/utils/NumberUtils.java

```diff
@@ -57,4 +57,11 @@ public static String getTimeLeft(int seconds) {
         return timeleft + seconds + "s";
     }
 
+    public static int getInt(String str, int defaultVal) {
+        try {
+            return Integer.parseInt(str);
+        } catch (NumberFormatException e) {
+            return defaultVal;
+        }
```

Comment:
ewwww, abusing try/catch for this is really unideal... We have a Pattern for Numerics you know?

Reply:
*roll eyes*